### PR TITLE
tox: run mypy

### PR DESCRIPTION
**Pull request by goneri**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1713

### CODE REVIEW

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
recheck

File: gouttelette/utils.py [hunk 1]
```diff
@@ -81,19 +85,21 @@ def python_type(value) -> str:
...
 class UtilsBase:
+    name: str
+
     def is_trusted(self, generator: str) -> bool:
-        if get_module_from_config(self.name, generator) is False:
-            print(f"- do not build: {self.name}")
-        else:
+        if get_module_from_config(self.name, generator):
             return True
+        else:
```
Comment (GomathiselviS) [inline gouttelette/utils.py]: else is not needed here

**Review by GomathiselviS — APPROVED** [review]

**Comment by goneri** [comment]
regate

**Comment by goneri** [comment]
regate

**Pull request merged**